bftengine/tests/bcstatetransfer/RVT_test.cpp

```diff
@@ -110,7 +153,281 @@ INSTANTIATE_TEST_CASE_P(validateRawValue,
                                           randomString(HASH_SIZE),
                                           randomString(HASH_SIZE)), );
 
+/////////////////// ending  of maths properties validation test for cryptoPP::Integer class ///////////////////////
+
+TEST_F(RVTTest, constructTreeWithSingleFirstNode) {
+  static constexpr uint32_t fetch_range_size = 4;
+  RangeValidationTree rvt(logging::getLogger("concord.bft.st.rvt"), RVT_K, fetch_range_size);
+  for (auto i = fetch_range_size; i <= fetch_range_size; i = i + fetch_range_size) {
+    STDigest digest(std::to_string(i).c_str());
+    rvt.addNode(i, digest.get(), BLOCK_DIGEST_SIZE);
+    ASSERT_TRUE(rvt.validate());
+  }
```

Comment:
This code can be seen in multiple tests, maybe it should be extracted possibly in a constexpr function?